ppr-ui/src/components/search/SearchBar.vue

```diff
@@ -30,7 +30,7 @@
       </v-col>
       <v-col align-self="end" cols="1" class="pl-3"/>
     </v-row>
-    <v-row>
+    <v-row v-if="typeOfSearch!=''">
```

Comment:
Would `v-if="typeOfSearch"` or `v-if="!!typeOfSearch"` work? Preferably one of these conditionals.

If not, you could keep the expression but i would use `!==` (type and value comparison operator) and also add some spacing.

Reply:
just fixed it!